### PR TITLE
[WIP] RPM hard limit depending on CLT #455

### PR DESCRIPTION
**Pull request by AlexHLinS**
added definitions for calibration table CLT x RPM limit

### CODE REVIEW

File: firmware/integration/rusefi_config.txt [hunk 1]
```diff
@@ -1397,7 +1398,12 @@ float[CLT_CURVE_SIZE] iacCoasting;    CLT-based idle position for coasting (used
 	linear_sensor_s highPressureFuel;
 	linear_sensor_s lowPressureFuel;
 
-uint8_t[536] unused3328;;"units", 1, 0, -20, 100, 0
+! issue 455: RPM hard limit depending on CLT
+int8_t[CLT_LIMITER_CURVE_SIZE] cltRevLimitRpmBins;CLT-based target RPM for hard limit depending on CLT LFAstyle;"C",        1,     0,   -70,    120,  0
+uint16_t[CLT_LIMITER_CURVE_SIZE] cltRevLimitRpm;See idleRpmPid;"",      1,     0,      0,    8000,   0
+
+! was [536] - CLT_LIMITER_CURVE_SIZE*size_of(int8_t) [4*1] - CLT_LIMITER_CURVE_SIZE*size_of(uint16_t) [4*2] = 536 - 4 - 8 = 524
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt:1405]: we don't need to record what the old line was here

File: firmware/integration/rusefi_config.txt [hunk 2]
```diff
@@ -1397,7 +1398,12 @@ float[CLT_CURVE_SIZE] iacCoasting;    CLT-based idle position for coasting (used
 	linear_sensor_s highPressureFuel;
 	linear_sensor_s lowPressureFuel;
 
-uint8_t[536] unused3328;;"units", 1, 0, -20, 100, 0
+! issue 455: RPM hard limit depending on CLT
+int8_t[CLT_LIMITER_CURVE_SIZE] cltRevLimitRpmBins;CLT-based target RPM for hard limit depending on CLT LFAstyle;"C",        1,     0,   -70,    120,  0
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt:1402]: what is `LFAstyle`?

**Comment by mck1117** [comment]
related to #455

File: firmware/integration/rusefi_config.txt (hunk 1 above)
Comment (AlexHLinS) [reply firmware/integration/rusefi_config.txt:1405]: Ok, will clear this comments

File: firmware/integration/rusefi_config.txt (hunk 2 above)
Comment (AlexHLinS) [reply firmware/integration/rusefi_config.txt:1402]: When this issuer is opens there is a comment "Variable RPM hard limit depending on engine temperate would be a nice feature right from Lexus LFA", so, I called this Lexus LFA style CLT Rev Limiter

Comment (mck1117) [reply firmware/integration/rusefi_config.txt:1402]: maybe `...hard limit depending on CLT, just like a Lexus LFA`

Comment (AlexHLinS) [reply firmware/integration/rusefi_config.txt:1402]: Ok, anyway I put the [WIP] tag, so will clear the old lines and change this comment on the final release of this ticket. If it possible.

Comment (mck1117) [reply firmware/integration/rusefi_config.txt:1402]: yes no problem - you can continue pushing to your branch and it'll just update the PR when you `git push`

**Comment by rusefillc** [comment]
todo Andrey: make two changes in master

**Pull request merged**